tests/periph_timer/Makefile

```diff
@@ -2,42 +2,4 @@ include ../Makefile.tests_common
 
 FEATURES_REQUIRED = periph_timer
 
-BOARDS_TIMER_25kHz := \
```

Comment:
BTW this should have been 250kHz :)

Reply:
> BTW this should have been 250kHz :)

Also noticed that and even [opened a PR](https://github.com/RIOT-OS/RIOT/pull/14807) until I noticed that #14799  was removing it...